cmd/tuf/app/init.go

```diff
@@ -73,14 +73,14 @@ func InitCmd(ctx context.Context, directory, previous, targets, snapshotRef stri
 
 	if previous == "" {
 		// Only initialize if no previous specified.
-		if err := repo.Init(false); err != nil {
+		if err := repo.Init(true); err != nil {
```

Comment:
worth mentioning what this boolean is for, or making it a named var for clarity?

Reply:
agreed, done!